plenum/test/replica/test_replica_addition.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from pytest import fixture
+
+from plenum.common.throughput_measurements import RevivalSpikeResistantEMAThroughputMeasurement
+from plenum.test.helper import sdk_send_random_and_check
+
+from plenum.test.pool_transactions.conftest import sdk_node_theta_added
+
+nodeCount = 6
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    old_throughput_measurement_class = tconf.throughput_measurement_class
+    old_throughput_measurement_params = tconf.throughput_measurement_params
+
+    tconf.throughput_measurement_class = RevivalSpikeResistantEMAThroughputMeasurement
+    tconf.throughput_measurement_params = {
+        'window_size': 5,
+        'min_cnt': 2
+    }
+
+    yield tconf
+
+    tconf.throughput_measurement_class = old_throughput_measurement_class
+    tconf.throughput_measurement_params = old_throughput_measurement_params
+
+
+def test_replica_addition(looper, sdk_pool_handle, txnPoolNodeSet,
```

Comment:
It's better to have a test name that shows the purpose of the test.
For example, `test_monitor_reset_after_replica_addition`.